traffic_ops/v4-client/servercheck.go

```diff
@@ -17,6 +17,8 @@ package client
 
 import (
 	"github.com/apache/trafficcontrol/lib/go-tc"
+	"net/http"
+	"net/url"
```

Comment:
standard library imports should come before project-local imports

Reply:
done.